### DISCUSSION TITLE
What are the built-in environment variables of Tauri?

### DISCUSSION THREAD

**Discussion by kjxbyz** [tauri-apps/tauri#7623] [Q&A] [answered]
What are the built-in environment variables of Tauri?

**Comment by FabianLars** [comment, marked as answer]
Do you mean the ones listed here? https://tauri.app/v1/api/config#buildconfig.beforedevcommand

If so, then these should be the same as the respective functions here https://tauri.app/v1/api/js/os#functions

**Reply by kjxbyz** [reply]
>Do you mean the ones listed here? https://tauri.app/v1/api/config#buildconfig.beforedevcommand

Yep.
